### ISSUE TITLE
Proposal: Split out the familial relations into a domain ontology

### ISSUE THREAD

**Issue by alanruttenberg** [CommonCoreOntology/CommonCoreOntologies#560]
There are a LOT of these - I stopped counting at 50 -looks like a third of all object properties are these - and they are so specific that I don't think they belong at the mid-level.

**Comment by alanruttenberg** [comment]
In addition
- They mostly don't have definitions, leaving the question of whether e.g. a step father is a valid value for 'has father', or an adopted person is a child - are these speaking to social roles or biological facts?
- Many should have disjointObjectProperty axioms, since I don't think you can be a mother and father at the same time (although maybe, if the steps, surrogates, and fatherness is considered a social role.

There's a tutorial about a familial relations ontology: https://oboacademy.github.io/obook/tutorial/fhkb/ that might serve as a basis for further enhancement if it becomes a domain ontology.

**Comment by johnbeve** [comment]
+1 
@mark-jensen I know this has been on your chopping block for a minute...
FYSA @APCox @CarterBeauBenson @neilotte @oliviahobai

**Comment by BrendaBraitling** [comment]
@alanruttenberg  @mark-jensen @johnbeve   I am not an expert at FOL but perhaps CCO could embrace a midlevel approach to kinship as presented in this paper - which you may be familiar with - It claims to handle kinship and be aligned to BFO2020-FOL.  It seems to be concise and immediately useful to real world systems as mentioned in the paper. I do not have the FOL expertise to evaluate it completely.  Interoperability across domains regarding information about humans seems critical.

[An Extendible Realism-Based Ontology for Kinship -2023 - Michael Rabenberg, Anuwat Pengput, Werner Ceusters](https://pmc.ncbi.nlm.nih.gov/articles/PMC11131162/pdf/nihms-1994678.pdf )

CCO probably should at least assert that a natural person will always be the biological child of its biological parents.  If the above proposal from the paper is not sufficient, the domains can then deal with the kinship beyond child - parent.

In general I think Midlevel is the best place to provide human biological filial generations to allow for interoperability across domains. This would aid in untangling domain specific, diverse configurations of family arrangements, households, legal parents vs biological parents, etc...

**Comment by alanruttenberg** [comment]
@BrendaBraitling Thanks for the reference! Werner is the person who has most used the BFO axiomatization other than myself. So the FOL will definitely be compatible.  It's undoubtedly well thought out and so worth paying attention to. 

Translating to OWL will be difficult in a number of cases, because of the ternary predicates - time indexing and others. But at least it is well defined. It looks like it is limited to "blood relations", so no accounting e.g for an adopted child, or a grandparent by a second marriage, or non-biological, but legal parents.

Regarding where to put the familial relations, some few might be appropriate, but not to the level that CCO has.  Having the relations in a domain ontology is no burden - just another import. But it's important, I think, to have CCO not be overwhelming large, and for a mid-level to exhibit some amount of consistency in how specific the terms get.  We have, e.g. Algorithm, with no subclasses (e.g why not sorting, function fitting, caching, visualization). Or Quality specification that specializes only to size and mass. We have no term for law. We have Biological sex, but no term for Gender - even the most general. So is it really appropriate that we have relation for third cousin? Even the cited paper only goes to first cousin. SNOMED CT, which the paper includes links to, doesn't have third cousin and it has something like 300,000 classes. Seems it would be rather uncommon for that relation to be used outside some specialized area. Also, it's complicated - you mention legal parents. We don't have those in CCO AFAICT. And there are all sorts of other complications. So that's why the suggestion it be made a domain ontology, beyond the simplest relations, maybe spouse and mother, father, daughter, son - by role, not biology. Or maybe just spouse, parent and child.

**Comment by CarterBeauBenson** [comment]
@johnbeve I agree. The next question is where should the relations go? They should remain publicly accessibly which limits us to CPO , the person ontology, linguistics ontology, or breaking the relations out into their own module.

**Comment by cameronmore** [comment]
I am in favor of both 1 and 2, and I would be happy to work on creating the new domain ontology. This would allow the opportunity to relook at those design patterns to see if there are any representations that could be improved and expand the ontology to include relevant classes as well.

**Comment by BrendaBraitling** [comment]
As @alanruttenberg mentions earlier on - this is very big pile of stuff.  Why not handle it rather than shovel it somewhere else?

In my mind, any computational system or human inquiry which seeks to use these elements only cares about the first filial relationship. The rest are computable and entirely dependent on this primary relationship. This is probably why we always think of extended families as a graph network construct called a family tree. 

Maude has a father Henry , Fred the Third has a father Fred the Second. Are they related? Obviously they are not directly related because they have different fathers...

We can traverse this family's graph network of object persons and their first hop child-parent/parent-child relationships

Traversal initiated at Fred 3 to discover if Maude is related and if so, how?

Fred 3 has parent Fred 2   Up 1 generation
Fred 2 has parent Fred1    Up 1 generation
Fred 1 has parent Henry   Up 1 generation
Henry has child Maude.    Down 1 generation

UP 3 generations and down one. 
Maude is in Fred 3's family tree. 
4 hops, 3 up, 1 down, second generation
Indirect relation as number of hops > generational hop

Anything beyond the child to parent/parent to child relationship is a computational exercise. This even holds true for genetic sequence similarities in DNA analysis.

**Would an ontology be required to provide all of those labels, like Great Aunt?**
Or, is this use case better served with a patterned nomenclature which can extend itself?
I would not want to define 'second cousin once removed' if I did not have to. It can be done.  **But then where does it end?**

https://www.reddit.com/r/coolguides/comments/retxrh/cousin_chart_with_of_dna_matches/

I would suggest a  human person has a parent relationship to another human person and a human person has a child - and the ontology has done its job. 

 And, to be more precise.... 

Each and every human has a parent. (actually two) 
If there is a parent relationship, then there is a child relationship. 
Not every human has a child.

The CCO as MLO could terminate at the human organism and then any domain ontology can easily extend.
Perhaps we do not need to move the pile elsewhere.

**Comment by alanruttenberg** [comment]
Arguably this is part of the domain of the person ontology.

**Comment by johnbeve** [comment]
@alanruttenberg agreed
